lightning/src/ln/peer_handler.rs

```diff
@@ -1911,31 +1911,31 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, OM: Deref, L: Deref, CM
 				for event in events_generated.drain(..) {
 					match event {
 						MessageSendEvent::SendAcceptChannel { ref node_id, ref msg } => {
-							log_debug!(self.logger, "Handling SendAcceptChannel event in peer_handler for node {} for channel {}",
+							log_debug!(WithContext {logger: &self.logger, peer_id: Some(*node_id), channel_id: None }, "Handling SendAcceptChannel event in peer_handler for node {} for channel {}",
```

Comment:
Some of these messages should contain a channel id, even if a temporary one, I'd imagine.